Source/HelixToolkit.Wpf/Visual3Ds/Composite/ViewCubeVisual3D.cs

```diff
@@ -30,6 +30,7 @@ public class ViewCubeVisual3D : ModelVisual3D
         public static readonly DependencyProperty BackTextProperty = DependencyProperty.Register(
             "BackText", typeof(string), typeof(ViewCubeVisual3D), new UIPropertyMetadata("B", (d,e)=> 
             {
+                var b = (d as ViewCubeVisual3D).GetCubefaceColor(1);
                 (d as ViewCubeVisual3D).UpdateCubefaceMaterial(1, Brushes.Red, e.NewValue == null ? "" : (string)e.NewValue);

```

Comment:
Should it be `(d as ViewCubeVisual3D).UpdateCubefaceMaterial(1, b, e.NewValue == null ? "" : (string)e.NewValue);`?